roadmap/implementers-guide/src/protocol-disputes.md

```diff
@@ -0,0 +1,74 @@
+# Disputes
+
+## Motivation
+
+All blocks that end up on chain should be valid.
+
+To assure attempts, successful or not, of including
+a block that is invalid in respect to the validation code, must therefore be handled in some way, disputes and their resolution are the formal process to resolve these situations.
+
+At some point a validator claims that the `PoV` (proof of validity) - which was distributed with candidate block - for a certain block is invalid.
+
+Now the dispute can be happening quite some time later than the inclusion, but also right during backing. As such the block is stored or more so it's solomon-reed encoded erasure chunks, from which the
```

Comment:
usually called Reed-Solomon

Reply:
```suggestion
Now the dispute can be happening quite some time later than the inclusion, but also right during backing. As such the block is stored or more so its Reed-Solomon encoded erasure chunks, from which the
```

Reply:
Don't ask why this is stuck the other way round in my head...